awx/main/models/schedules.py

```diff
@@ -35,6 +35,64 @@
 UTC_TIMEZONES = {x: tzutc() for x in dateutil.parser.parserinfo().UTCZONE}
 
 
+def _assert_timezone_id_is_valid(rrules) -> None:
+    broken_rrules = [str(rrule) for rrule in rrules if rrule._dtstart and rrule._dtstart.tzinfo is None]
+    if not broken_rrules:
+        return
+    raise ValueError(
+        'The following rrules do not have a valid timezone: ' f'{broken_rrules}. Set a valid timezone ' '(e.g., America/New_York).',
```

Comment:
Looks like the formatter did the dumb thing and inlined all the implicit concat.
```suggestion
        f'The following rrules do not have a valid timezone: {broken_rrules}. Set a valid timezone (e.g., America/New_York).',
```